packages/e2e-test-utils-playwright/src/editor/open-document-settings-sidebar.ts

```diff
@@ -5,14 +5,18 @@ import type { Editor } from './index';
 import { expect } from '../test';
 
 /**
- * Clicks on the button in the header which opens Document Settings sidebar when it is closed.
+ * Clicks on the button in the header which opens Document Settings sidebar when
+ * it is closed.
  *
  * @param {Editor} this
  */
 export async function openDocumentSettingsSidebar( this: Editor ) {
```

Comment:
I think we have a similar helper in Puppeteer utils. Should we update it as well?

Reply:
Good point! Updated in https://github.com/WordPress/gutenberg/pull/47498/commits/725ad50f5c0bfcd46f646f150de0171374f623dd.